.travis.yml

```diff
@@ -1,6 +1,9 @@
 language: python
 python:
   - "2.7"
+  - "3.4"
```

Comment:
I don't think this is real. https://github.com/dimagi/touchforms/blob/master/run_tests.sh uses jython to actually run the code. jython 3 is in development

Reply:
ah I see